packages/color-modes/src/custom-properties.ts

```diff
@@ -1,8 +1,9 @@
 import { css, get, Theme, __internalGetUseRootStyles } from '@theme-ui/css'
 
-const toVarName = (key: string) => `--theme-ui-${key}`
+const stripKeyDefault = (key: string) => key.replace('-__default', '')
+const toVarName = (key: string) => `--theme-ui-${stripKeyDefault(key)}`
 const toVarValue = (key: string, value: string | number) =>
-  `var(${toVarName(key)}, ${value})`
+  `var(${toVarName(stripKeyDefault(key))}, ${value})`
```

Comment:
I don't get this part.

Isn't `toVarValue` now

```
const toVarValue = (key: string, value: string | number) =>
  `var(`--theme-ui-${stripKeyDefault((stripKeyDefault(key)))`}, ${value})`
```

stripKeyDefault is called twice.

Reply:
You're right, I did this pretty quickly I didn't realize they were fed into each other.
Just fixed it in a new commit.